vcx/libvcx/src/api/vcx.rs

```diff
@@ -308,15 +308,50 @@ pub extern fn vcx_update_institution_info(name: *const c_char, logo_url: *const
     error::SUCCESS.code_num
 }
 
+/// Update agency webhook url setting
+///
+/// #Params
+///
+/// command_handle: command handle to map callback to user context.
+///
+/// notification_webhook_url: URL to which the notifications should be sent
+///
+/// cb: Callback that provides error code of the result
+///
+/// #Returns
+/// Error code as u32
 #[no_mangle]
-pub extern fn vcx_update_webhook_url(notification_webhook_url: *const c_char) -> u32 {
-    info!("vcx_update_webhook >>>");
+pub extern fn vcx_update_webhook_url(command_handle: CommandHandle,
```

Comment:
Please pay attention that this changes are "breaking changes" from API point of view. I think they are reasonable in this particular place as this functionality is pretty new, but we should be careful with breaking changes in general.

Reply:
OK, point taken.